sdk/communication/Azure.Communication.Email/src/Models/EmailRecipients.cs

```diff
@@ -22,10 +22,10 @@ public partial class EmailRecipients
         /// <param name="bcc">Email bcc recipients. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="to"/> is null. </exception>
         public EmailRecipients(
```

Comment:
Feels like we should remove the constructor for EmailRecipients in the generated file.
I'm not a 100% sure how to do that without changing the SWAGGER.

Are we not going to change the SWAGGER to not make To not required?

Reply:
Yeah, we are going to update the swagger. Komivi mentioned that there are other changes coming too so plan is to update all of them together. This change is a temporary workaround to unblock customers.